packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -731,7 +731,7 @@ export type SdkExampleValue =
   | SdkUnionExampleValue
   | SdkModelExampleValue;
 
-export interface SdkExampleValueBase {
+interface SdkExampleValueBase {
```

Comment:
this is not meant to be exported, others would not have been using, therefore despite this is a breaking but I think this should have no impact.

Reply:
Unsure about this. For Java, the base class must be visible. But this is TS.